pom.xml

```diff
@@ -797,6 +797,10 @@
                         <groupId>commons-logging</groupId>
                         <artifactId>commons-logging</artifactId>
                     </exclusion>
+                    <exclusion>
```

Comment:
Could we please add a comment with the exclusion before the final PR?

In the future this will look like someone was just keeping the dependencies tidy so could easily be deleted if new tests need it.